src/test/kotlin/apply/application/TermServiceTest.kt

```diff
@@ -0,0 +1,96 @@
+package apply.application
+
+import apply.domain.recruitment.RecruitmentRepository
+import apply.domain.term.Term
+import apply.domain.term.TermRepository
+import io.mockk.Runs
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.just
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertAll
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
+import support.test.UnitTest
+
+@UnitTest
+internal class TermServiceTest {
+    @MockK
+    lateinit var termRepository: TermRepository
+
+    @MockK
+    lateinit var recruitmentRepository: RecruitmentRepository
+
+    private lateinit var termService: TermService
+
+    @BeforeEach
+    internal fun setUp() {
+        termService = TermService(termRepository, recruitmentRepository)
+    }
+
+    @Test
+    fun `기수를 생성한다`() {
+        every { termRepository.save(any()) } returns Term("3기")
+        every { termRepository.existsByName(any()) } returns false
+        assertDoesNotThrow { termService.save(TermData("3기")) }
+    }
+
+    @Test
+    fun `기본 기수명으로 기수를 생성할 수 없다`() {
+        val request = TermData(Term.SINGLE.name)
+        assertThrows<IllegalStateException> { termService.save(request) }
+    }
+
+    @Test
+    fun `중복된 기수명으로 기수를 생성할 수 없다`() {
+        every { termRepository.existsByName(any()) } returns true
+        assertThrows<IllegalStateException> { termService.save(TermData("3기")) }
+    }
+
+    @Test
+    fun `기수를 조회한다`() {
+        val terms = listOf(Term("3기"), Term("4기"))
+        every { termRepository.findAll() } returns terms
+        val responses = termService.findAll()
+        assertAll(
+            { assertThat(responses).hasSize(3) },
+            {
+                assertThat(responses).containsExactlyInAnyOrder(
+                    TermResponse(Term.SINGLE),
+                    TermResponse(terms[0]),
+                    TermResponse(terms[1])
+                )
+            }
+        )
```

Comment:
`hasSize()`와 `containsExactlyInAnyOrder()`는 중복되지 않을까요?

```suggestion
        assertThat(responses).containsExactlyInAnyOrder(
            TermResponse(Term.SINGLE),
            TermResponse(terms[0]),
            TermResponse(terms[1])
        )
```